src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -36,14 +37,8 @@
 {
     private static TrackedList<BiomeEntry>[] biomes = setupBiomes();
 
-    public static List<Biome> oceanBiomes = new ArrayList<Biome>();
-
-    static
-    {
-        oceanBiomes.add(Biomes.OCEAN);
-        oceanBiomes.add(Biomes.DEEP_OCEAN);
-        oceanBiomes.add(Biomes.FROZEN_OCEAN);
-    }
+    private static List<Biome> deepOceanBiomes = new ArrayList<Biome>();
```

Comment:
Why are deep ocean biomes special cased? If you already add something special for deep oceans, why not for "normal" oceans as well?

Reply:
Accidently left that part over from a different way I attempted doing the ocean biome entry system.

Will fix 👍